blog/2025-08-26-verifying-bls12-on-ethereum.md

```diff
@@ -0,0 +1,40 @@
+---
+title: "Verifying the `quicknet` beacons on Ethereum"
+description: "At long last!"
+date: 2025-08-26
+author: Julie Bettens
```

Comment:
```suggestion
author: Julie Tyché Bettens
```

Tyche being the Greek deity of fortune and fate, I feel like my second given name is à propos here